tests/expectations/test_run_diagnostics_supporting_methods.py

```diff
@@ -186,6 +186,9 @@ def test__get_execution_engine_diagnostics_with_one_metrics_diagnostics():
 
 
 ### Tests for _get_test_results
+@pytest.mark.skip(
```

Comment:
@kenwade4 Is it acceptable to skip the test?  Should we instead try to make the Docker container you added also work in Azure and then actually run these tests?  If I recall, @Shinnnyshinshin has done work on Azure containers.  Thank you.